### ISSUE TITLE
Reload the saved PMML model and run inference in Python

### ISSUE THREAD

**Issue by nicolas-gervais** [jpmml/sklearn2pmml#340]
I successfully trained the model with `sklearn2pmml`. 

Is it possible to reload it in Python for inference? Something like:

```
import sklearn2pmml

model = sklearn2pmml.fromFile('DecisionTreeIris.pmml')

result = model.predict({
    "Sepal_Length" : 5.1,
    "Sepal_Width" : 3.5,
    "Petal_Length" : 1.4,
    "Petal_Width" : 0.2
})
```

**Comment by vruusmann** [comment]
The JPMML family of software has two product verticals. First, the "conversion vertical" is based on the [JPMML-Converter](https://github.com/jpmml/jpmml-converter) library and its descendants such as JPMML-Python, JPMML-SkLearn and eventually SkLearn2PMML. Second, the "evaluation vertical" is based on the [JPMML-Evaluator](https://github.com/jpmml/jpmml-evaluator) library.

In the current case, you must be looking for the "[JPMML-Evaluator for Python](https://github.com/jpmml/jpmml-evaluator-python)", which is available as the `jpmml_evaluator` package in PyPI.

By using JPMML family libraries throughout all workflow steps, you can have a reasonable expectation that things work and perform as advertised. If you replace "JPMML-Evaluator for Python" with eg. PyPMML, then things can happen, and I won't be wasting my time debugging these.

TLDR: I just published a small article in Openscoring.IO blog about extending Scikit-Learn with CHAID estimators:
https://openscoring.io/blog/2022/07/14/sklearn_chaid_pmml/

The second part of the article is all about scoring PMML models with "JPMML-Evaluator for Python". Do check it out!
